pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -84,16 +84,27 @@ var _ = Describe("Pod Network", func() {
 	var mtu int
 	var cacheFactory cache.InterfaceCacheFactory
 	var libvirtUser string
-	var podnic podNICImpl
-
+	var newPodNIC = func(vmi *v1.VirtualMachineInstance) podNIC {
+		return podNIC{
+			cacheFactory: cacheFactory,
+			handler:      mockNetwork,
+			vmi:          vmi,
+		}
+	}
+	var defaultPodNIC = func(vmi *v1.VirtualMachineInstance) podNIC {
```

Comment:
It would be easier to understand it is a func if it had a verb prefix - `createDefaultPodNIC`

Reply:
Done